examples/speaker_recognition/conf/config.yaml

```diff
@@ -7,17 +7,25 @@ n_filters: &n_filters 512
 
 model:
   train_ds:
-    manifest_filepath: '/data/manifests/voxceleb1/train/train_manifest.json'
+    manifest_filepath: '/data/an4_speaker/train.json'
```

Comment:
Use ??? here and pass path explicitly.

Reply:
that is what I was thinking but having some root_dir passed to model in cfg, is something i'm not comfortable with.

Reply:
I don't understand the issue maybe, The rest of the models follow the same approach.